cmd/orphan-finder/main.go

```diff
@@ -238,7 +340,22 @@ func main() {
 			Status:  string(core.OCSPStatusGood),
 		})
 		cmd.FailOnError(err, "Generating OCSP")
-		_, err = sa.AddCertificate(ctx, der, int64(*regID), ocspResponse, &issuedDate)
+
+		switch typ {
+		case certOrphan:
+			_, err = sa.AddCertificate(ctx, der, int64(*regID), ocspResponse, &issuedDate)
```

Comment:
super nit: instead of needing to cast this id everywhere, you could use `flag.Int64` instead of `flag.Int` and get what you want out of the box.

Reply:
Makes sense, resolved in 0e8757e